src/components/author/author.twig

```diff
@@ -7,17 +7,30 @@
     {% endblock %}
   {% endembed %}
   <div class="o-media__content">
-    <div>
-      <span class="u-hidden-visually">Authored by</span>
+    <p>
+      <span class="u-hidden-visually">By</span>
       {% for author in authors %}
         {% if loop.last and loop.length > 1 %}and {% endif %}
         <a href="{{ author.link }}">{{ author.name }}</a>
         {%- if not loop.last and loop.length > 2 %},{% endif %}
       {% endfor %}
-    </div>
-    <div>
-      <span class="u-hidden-visually">Published on</span>
-      <time datetime="{{ date|date('Y-m-d') }}">{{ date|date('F jS, Y') }}</time>
-    </div>
+    </p>
+
+    {% if date is defined %}
+      <p>
+        <span class="u-hidden-visually">
+          Published on {{ date|date('F jS, Y') }}
+        </span>
+        <time aria-hidden="true" datetime="{{ date|date('Y-m-d') }}">
+          {{ date|date('F jS, Y') }}
+        </time>
+      </p>
```

Comment:
When testing with macOS VoiceOver, the date would get read twice before, which was a confusing experience.

A bit of research led me to [this tweet](https://twitter.com/LeonieWatson/status/1333078194925264898) stating:
> AFAIK `<time datetime>` is not well accessibility supported by browsers/screen readers.

With this in mind, I decided to make the screen reader experience better while still keeping the `<time>` but hiding it from the screen reader. I kept the `<time>` element because per the [MDN Web Docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time), it allows for better search engine results (though, admittingly, I don't know much about this):
> The `<time>` HTML element represents a specific period in time. It may include the `datetime` attribute to translate dates into machine-readable format, allowing for better search engine results or custom features such as reminders.

Reply:
Your context is great, but I wish more of it would be included as a Twig comment within the template itself so this is as clear in the future as it is while reviewing this PR.

Reply:
Done! https://github.com/cloudfour/cloudfour.com-patterns/pull/1607/commits/63349fd2f8c6230fc1e3de84181068f9a0517ac1